src/ecosystem/corals/growth.jl

```diff
@@ -318,7 +318,13 @@ function _shift_distributions!(cover::SubArray, growth_rate::SubArray, dist_t::S
     μ = mean(dist_t[1])
     dist_t[2] = truncated(Normal(μ, stdev[2]), minimum(dist_t[1]), μ + HEAT_UB)
 
-    return nothing
+    # Calculate weights
+    w1::Vector{Float64} = replace!(c_increase ./ c_t1, NaN => 0.0)
```

Comment:
I'm getting an error here because c_increase isn't defined. I tried defining at ct_t-ct but still getting missing variables error. I'm guessing something needs to be copied from the heritability code.

Reply:
Hmm, faulty rebase I think. Let me fix.

Reply:
Should be all fixed, please try now.

Reply:
I just pulled and still get the same error...The pull only seemed to update some comments in growth.jl

Reply:
> I just pulled and still get the same error...The pull only seemed to update some comments in growth.jl

??? That's certainly not the case....

You don't get anything else like an error message?